packages/ui-components/src/Tooltip/Tooltip.scss

```diff
@@ -0,0 +1,218 @@
+@import "../styles/tokens.scss";
+
+//
+// Tooltips
+// --------------------------------------------------
+
+$border-radius-base: 3px;
+$tooltip-border-width: 1px;
+$tooltip-shadow-width: 0px;
+$tooltip-arrow-width: 6px;
+
+@mixin crl-tooltip-style( $bg, $border, $color ) {
+  .crl-tooltip-inner {
+    border-color: $border;
+    background-color: $bg;
+    color: $color;
+  }
+  .crl-tooltip-arrow{
+    border-color: $border;
+  }
+  .crl-tooltip-arrow-inner {
+    border-color: $bg;
+  }
+}
+
+//
+// Default style
+// --------------------------------------------------
+.default {
```

Comment:
Since this isn't using css modules these styles will be global and could also cause conflicts in consuming apps.

Reply:
`:global` namespace not working good with current styles due to a bit tricky way of styling tooltip arrow and provide easy to adjust theme classes. but I moved that definitions inside `.crl-tooltip` namespace to minify possible conflicts